### PR TITLE
SKY130_PDK

### PR DESCRIPTION
**Pull request by sethupathib**
Added PDK primitives for SKY130

### CODE REVIEW

**Comment by sethupathib** [comment]
I'm not quite sure about the primitive.py file. We might need to have a discussion.
SKY130 seems to have different flavours of cells like lvt,hvt etc. (Which I had commented earlier.)

File: pdks/SKY130_PDK/primitive.py [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+
+#In the layers.json file "Fin" is mapped to "active". Is this how it is supposed to be?
+#I am having some doubts here.
+
+
+
+
+
+
+class MOSGenerator(default.MOSGenerator):
```
Comment (parijatm) [inline pdks/SKY130_PDK/primitive.py:39]: Don't inherit from default.MOSGenerator since you are effectively reimplementing everything in it.

Can you summarize what changes you need to make from a layer perspective? Its also possible that simply changing layers.json may be enough.

Comment (parijatm) [reply pdks/SKY130_PDK/primitive.py:39]: @arvuce22 can help with this exercise too.

**Comment by arvuce22** [comment]
@thesourcerer8 <https://github.com/thesourcerer8> we can set up a meeting
to discuss these issues.

**Comment by sethupathib** [comment]
@arvuce22 @parijatm : We were going through some of the ALIGN literature to understand the tool better. 
@thesourcerer8 will be initiating a meeting very soon.
